### PR TITLE
feat(dbt): adds source and staging models for arb1 events

### PR DESCRIPTION
**Pull request by ccerv1**

### CODE REVIEW

File: warehouse/dbt/models/intermediate/blockchain_events/int_arbitrum_one_contract_invocation_events.sql [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    },
+    unique_id="id",
+    on_schema_change="append_new_columns",
+    incremental_strategy="insert_overwrite"
+  )
+}}
+{% if is_incremental() %}
+  {% set start = "TIMESTAMP_SUB(_dbt_max_partition, INTERVAL 1 DAY)" %}
+{% else %}
+  {% set start = "'1970-01-01'" %}
```
Comment (ccerv1) [inline warehouse/dbt/models/intermediate/blockchain_events/int_arbitrum_one_contract_invocation_events.sql:17]: @ravenac95 check for gas logic

File: warehouse/dbt/models/intermediate/blockchain_events/int_arbitrum_one_traces.sql [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+      "field": "block_timestamp",
+      "data_type": "timestamp",
+      "granularity": "day",
+    },
+    unique_id="id",
+    on_schema_change="append_new_columns",
+    incremental_strategy="insert_overwrite"
+  )
+}}
+{{ filtered_blockchain_events("BASE", "base", "traces") }}
```
Comment (ravenac95) [inline warehouse/dbt/models/intermediate/blockchain_events/int_arbitrum_one_traces.sql:14]:
```suggestion
{{ filtered_blockchain_events("ARBITRUM_ONE", "arbitrum_one", "traces") }}
```

File: warehouse/dbt/models/intermediate/blockchain_events/int_arbitrum_one_transactions.sql [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+      "field": "block_timestamp",
+      "data_type": "timestamp",
+      "granularity": "day",
+    },
+    unique_id="id",
+    on_schema_change="append_new_columns",
+    incremental_strategy="insert_overwrite"
+  )
+}}
+{{ filtered_blockchain_events("arbitrum_one", "base", "transactions") }}
```
Comment (ravenac95) [inline warehouse/dbt/models/intermediate/blockchain_events/int_arbitrum_one_transactions.sql:14]:
```suggestion
{{ filtered_blockchain_events("arbitrum_one", "arbitrum_one", "transactions") }}
```

Comment (ccerv1) [reply warehouse/dbt/models/intermediate/blockchain_events/int_arbitrum_one_transactions.sql:14]: @yeah, was just noticing this!

**Comment by ravenac95** [comment]
@ccerv1 we have a weird chicken + egg situation for the playground but we'll need to update the base_playground and playground sources to include references for arb1

**Comment by ravenac95** [comment]
Hrm for the type conversion it seems we will need to do this: https://stackoverflow.com/questions/70467412/bigquery-cast-hex-string-to-numeric-or-bignumeric

**Pull request merged**